spec/facades/photo_facade_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Photo Facade' do
+  describe 'Photo', :vcr do
+    it 'makes an API call using the Unsplash gem' do
```

Comment:
consider: the it statement can make reference to the fact that multiple photos are returned